src/onboarding/Configure/Setup.js

```diff
@@ -10,7 +10,7 @@ import { OrgTemplateType } from '../../prop-types'
 export const CONFIGURE_MODE_SELECT = Symbol('CONFIGURE_MODE_SELECT')
 export const CONFIGURE_MODE_CONFIGURE = Symbol('CONFIGURE_MODE_CONFIGURE')
```

Comment:
We should also probably update these to start with `SETUP_MODE_`, and also:

- Rename `ConfigureStepsPanel` to become `SetupStepsPanel`.
- Rename the directory to become `Setup/`?

Reply:
> We should also probably update these to start with `SETUP_MODE_`, and also:
> 
> * Rename `ConfigureStepsPanel` to become `SetupStepsPanel`.
> * Rename the directory to become `Setup/`?

Yes sure! makes sense indeed